### PR TITLE
installing git-lfs and downloading large files on travis

### PR DESCRIPTION
**Pull request by lbergelson**
adding a test for large files existance in the "large" test group

large files should be available for tests on travis after this pull request

### CODE REVIEW

File: scripts/install_git_lfs.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+echo "cloning"
+git clone https://github.com/github/git-lfs.git
+
+echo "cd into git-lfs"
+cd git-lfs
+
+echo "checkout commit e8a5cfb85e27c1284a73a2eb8269bd5f5c4d0955"
+git checkout v0.6.0
```
Comment (droazen) [inline scripts/install_git_lfs.sh]: Declare version as shell variable at top

File: src/test/resources/large/.gitattributes [hunk 2]
```diff
@@ -6,3 +6,4 @@ dbsnp_138.b37.20.21.vcf.idx filter=lfs diff=lfs merge=lfs -text
 human_g1k_v37.20.21.dict filter=lfs diff=lfs merge=lfs -text
 human_g1k_v37.20.21.fasta filter=lfs diff=lfs merge=lfs -text
 human_g1k_v37.20.21.fasta.fai filter=lfs diff=lfs merge=lfs -text
+example_lfs_file.txt filter=lfs diff=lfs merge=lfs -text
```
Comment (droazen) [inline src/test/resources/large/.gitattributes]: Wrong file?

File: src/test/resources/large/exampleLargeFile.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+This is a file to test if git-lfs is working.  Please don't change this text.
```
Comment (droazen) [inline src/test/resources/large/exampleLargeFile.txt]: You checked in the contents? So this is not being tracked by git lfs

**Comment by droazen** [comment]
Review complete -- back to @lbergelson

**Comment by droazen** [comment]
One additional comment: can you please update the hellbender README to mention the necessity of running git lfs pull?

**Comment by lbergelson** [comment]
@droazen Back to you.  Should be fixed.

File: README.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ Requirements
 
 * Git 2.5
 
-* The [git-lfs plugin](https://git-lfs.github.com/). Run `git lfs init` after installing.
+* The [git-lfs plugin](https://git-lfs.github.com/). Run `git lfs init` after installing, followed by `git lfs pull` to download the large files.  We recommend v0.6.0 but newer version may also work.
```
Comment (droazen) [inline README.md]: version -> versions

**Comment by droazen** [comment]
Back to @lbergelson -- one comment then merge away

**Pull request merged**